tests/lib/hardware/openstack_libcloud.py

```diff
@@ -233,6 +233,20 @@ def get_connection(self):
         )
         return connection
 
+    def _get_image(self, identifier=None):
+        try:
+            return self._get_image_by_id(identifier)
+        except libcloud.common.exceptions.BaseHTTPError:
+            logger.debug('No image found by id.
```

Comment:
You'll need to close the string on this line. It'll concat the next one directly to it though so you'll also need to leave a whitespace either at the end of this one or the beginning of the next one.

Reply:
Ok sorry about that